handlers/challenge_handler.py

```diff
@@ -865,12 +864,11 @@ def update_ctf_purpose(slack_wrapper, ctf):
         Update the purpose for the ctf channel.
         """
         purpose = dict(ChallengeHandler.CTF_PURPOSE)
-        purpose["ota_bot"] = "DO_NOT_DELETE_THIS"
+        purpose["ota_bot"] = "OTABOT"
```

Comment:
Do we need this field at all if we're going to remove the warning?

Reply:
I'd say it's not needed anymore, if we are going to remove it.

Reply:
We currently use the "otabot" key to identify the channels managed by bishop. 

We could instead check, if the purpose contains `"type"` and equals `CTF` or `CHALLENGE`, but I would prefer to keep the key for this, since it's the only reference at all in the bot to its origin.